tomcat/src/main/java/org/apache/coyote/http11/response/ResponseBody.java

```diff
@@ -0,0 +1,18 @@
+package org.apache.coyote.http11.response;
+
+public class ResponseBody {
+
+    private String responseBody;
+
+    public ResponseBody() {
+        this.responseBody = "";
+    }
```

Comment:
```suggestion
    public static ResponseBody empty() {
        return new ResponseBody("");
    }
```
이런 코드가 휴먼 에러를 방지할 것 같은데 정적 팩토리 메서드로 확실하게 네이밍을 해주는 것에 대해 어떻게 생각하시나요

Reply:
명확한 네이밍을 주는게 휴먼 에러를 방지할 수 있겠네요!